<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings_dialog</class>
 <widget class="QDialog" name="Settings_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jarrImageViewer.ico</normaloff>jarrImageViewer.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="dir1Label">
       <property name="text">
        <string>Directory 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dir1Line"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectDir1Button">
       <property name="text">
        <string>Select ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="dir2Label">
       <property name="text">
        <string>Directory 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dir2Line"/>
     </item>
     <item>
      <widget class="QPushButton" name="selectDir2Button">
       <property name="text">
        <string>Select ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="maxHeightLabel">
       <property name="text">
        <string>Max Height: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxHeightSpinBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxWidthLabel">
       <property name="text">
        <string>Max Width: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxWidthSpinBox">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="orderByLabel">
       <property name="text">
        <string>Order by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orderByComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Date</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="randomLabel">
       <property name="text">
        <string>Random Order: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="randomCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="slideShowTime">
       <property name="text">
        <string>Slideshow Time (secs):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="slideTimeSpinBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="labelShowDebug">
       <property name="text">
        <string>Show Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showDebugCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="win_x_offset">
       <property name="text">
        <string>Window X offset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="win_x_offset_spinBox">
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="win_y_offset">
       <property name="text">
        <string>window Y offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="win_y_offset_spinBox">
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
